### PR TITLE
Result view

### PR DESCRIPTION
**Pull request by lorenzoPrimi**
I guess the new ooni-wui works good now. Added possibility to select multiple tests in the webview.
I need this branch to be merged to start to integrate the Web Connectivity test

### CODE REVIEW

File: NetProbe/View/ResultViewController.h [hunk 1]
```diff
@@ -0,0 +1,17 @@
+//
+//  ResultViewController.h
+//  NetProbe
+//
+//  Created by Lorenzo Primiterra on 22/07/16.
+//  Copyright © 2016 Simone Basso. All rights reserved.
+//
```
Comment (bassosimone) [inline NetProbe/View/ResultViewController.h]: Can you use here the standard include used in other MK files, please?

File: NetProbe/View/ResultViewController.m [hunk 2]
```diff
@@ -0,0 +1,118 @@
+//
+//  ResultViewController.m
+//  NetProbe
+//
+//  Created by Lorenzo Primiterra on 22/07/16.
+//  Copyright © 2016 Simone Basso. All rights reserved.
+//
```
Comment (bassosimone) [inline NetProbe/View/ResultViewController.m]: Can you use here the standard include used in other MK files, please?

File: NetProbe/View/ResultViewController.m [hunk 3]
```diff
@@ -0,0 +1,118 @@
...
+
+@end
+
+@implementation ResultViewController
+
+- (void)viewDidLoad {
+    [super viewDidLoad];
+    NSArray *items = [self getItems];
+    if ([items count] > 1){
+        UIBarButtonItem *chiudi = [[UIBarButtonItem alloc] initWithBarButtonSystemItem:UIBarButtonSystemItemAction target:self action:@selector(selectTest)];
```
Comment (bassosimone) [inline NetProbe/View/ResultViewController.m]: Wouldn't it be better to call the variable `close`?

File: NetProbe/View/ResultViewController.m [hunk 4]
```diff
@@ -0,0 +1,118 @@
...
+
+-(void) loadScreen :(NSString*) content{
+    WKWebViewConfiguration *configuration = [[WKWebViewConfiguration alloc] init];
+    WKUserContentController* userController = [[WKUserContentController alloc] init];
+    
+    
+    NSString* MeasurementJSON = [NSString stringWithFormat:@"\n var MeasurementJSON = {  \n"
+                                 "get: function() {  \n"
+                                 "return "
+                                 //"'%@';"
```
Comment (bassosimone) [inline NetProbe/View/ResultViewController.m]: What is the purpose of the commented out code?

File: NetProbe/View/ResultViewController.m [hunk 5]
```diff
@@ -0,0 +1,118 @@
...
+        [self loadScreen:[[self getItems] objectAtIndex:buttonIndex]];
+    }
+}
+
+- (void)didReceiveMemoryWarning {
+    [super didReceiveMemoryWarning];
+    // Dispose of any resources that can be recreated.
+}
+
+/*
```
Comment (bassosimone) [inline NetProbe/View/ResultViewController.m]: Can we get rid of commented out code or is there a reason to keep it around?

File: NetProbe/View/SettingsTableViewController.m [hunk 6]
```diff
@@ -58,11 +59,18 @@ - (UITableViewCell *)tableView:(UITableView *)tableView cellForRowAtIndexPath:(N
         cell.accessoryView = switchview;
     }
     else {
+        cell = [tableView dequeueReusableCellWithIdentifier:@"CellSub" forIndexPath:indexPath];
+        NSString *current = [settingsItems objectAtIndex:indexPath.row];
+        cell.textLabel.text = NSLocalizedString(current, nil);
+        cell.detailTextLabel.text = [[NSUserDefaults standardUserDefaults] objectForKey:current];
+    }
+    /*
+    else {
```
Comment (bassosimone) [inline NetProbe/View/SettingsTableViewController.m]: Is there a reason for keeping this code around?

**Comment by bassosimone** [comment]
Thank you @lorenzoPrimi! I am already testing the application on the emulator.

Was wondering: shouldn't this PR also remove the custom code to pretty print JSON?

**Comment by bassosimone** [comment]
Also, perhaps the time of the measurement should be converted to local time?

**Comment by bassosimone** [comment]
The measurements results do not seem to be correct however: too much information is missing in the `test_keys` field of `tcp-connect` and `dns-injection`. Also for `http-invalid-request-line` I think I can only see the first of the four exchanges with the backend.

**Comment by bassosimone** [comment]
It would be useful to be able to copy and paste the JSON. Perhaps, it's just me who is, er, just a little dumb, but currently I can't do that for publishing here the results I see.

File: NetProbe/View/ResultViewController.h (hunk 1 above)
Comment (lorenzoPrimi) [inline NetProbe/View/ResultViewController.h]: Changed it everywhere

File: NetProbe/View/ResultViewController.m (hunk 4 above)
Comment (lorenzoPrimi) [inline NetProbe/View/ResultViewController.m]: It was the previous code to inject the JSON, I'll remove it

File: NetProbe/View/SettingsTableViewController.m (hunk 6 above)
Comment (lorenzoPrimi) [inline NetProbe/View/SettingsTableViewController.m]: Yes, I want to keep this until we decide how to finally implement settings. At the moment I removed the Advanced settings screen as we have only three items

**Comment by lorenzoPrimi** [comment]
The local time and everything you see in the result screen it part of ooni-wui, not this repo.

**Comment by bassosimone** [comment]
Good, then, I am going to merge this! 👍

**Pull request merged**